gov/staker/reward_calculation.gno

```diff
@@ -32,11 +33,27 @@ func currentBalance() uint64 {
 	return currentGNSBalance
 }
 
+func getCurrentProtocolFeeBalance() map[string]uint64 {
+	gotAccuProtocolFee := pf.GetAccuTransferToGovStaker()
```

Comment:
maybe missing `pf.DistributeProtocolFee()` ??
or remove this function and uncomment L#23